.github/workflows/cherryPick.yml

```diff
@@ -11,6 +11,9 @@ on:
         required: false
         default: ''
 
+env:
```

Comment:
Is this required to use `gh` cli? Doesn't this provide the `GITHUB_TOKEN` to all steps? Could we just provide it to those that use `gh` cli?

Reply:
> Is this required to use gh cli?

Yep, a `GITHUB_TOKEN` is required.

> Doesn't this provide the GITHUB_TOKEN to all steps?

Yep.

> Could we just provide it to those that use gh cli?

We _could_, I just thought that this would be a more convenient way to DRY it up. The only community actions still used in these workflows are already provided a github token.

Reply:
Right, but if someone adds a new step without understanding that, then they could get a token on accident right?

Reply:
Correct. I personally don't have much of a concern here but if you think it's a security best practice to only provide the token at the step-level I can make that change.